source/quick-start-sinatra/configure-mongodb.txt

```diff
@@ -0,0 +1,52 @@
+.. _mongoid-quick-start-sinatra-connect-to-mongodb:
+
+=================================
+Configure Your MongoDB Connection
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Specify target database in connection string
+      
+      When connecting to an Atlas cluster, you must specify the database that
+      you want to interact with as the default database in your connection string.
+      You must add the database name to your connection string **after the hostname**.
+      
+      We want to access the ``sample_restaurants`` database for this tutorial.
+      For example, if the connection string for your Atlas cluster is
+      ``"mongodb+srv://user0:pass123@mongo0.example.com/"``, you can
+      specify the target database as shown in the following code:
+
+      .. code-block:: none
+
+         mongodb+srv://user0:pass123@mongo0.example.com/sample_restaurants
```

Comment:
```suggestion
         mongodb+srv://<username>:<password>@mongo0.example.com/sample_restaurants
```

Reply:
I'll leave the connection string as is an specify that its a sample